src/lib/components/common/Input.svelte

```diff
@@ -0,0 +1,7 @@
+<script>
+</script>
+
+<input
```

Comment:
@SankhajaH You have to add logic to bind the input value to the parent. Define a `value` prop like this https://svelte.dev/tutorial/text-inputs and export it like https://svelte.dev/tutorial/declaring-props

Reply:
@NetworkCable will do! Thanks for the info